bitly/src/bitly_short_link_handler.erl

```diff
@@ -0,0 +1,30 @@
+-module(bitly_short_link_handler).
+
+-export([init/2]).
+
+init(Req, State) ->
+    Url = get_request_url(Req),
+    case cowboy_req:method(Req) of
+        <<"POST">> ->
+            {Status, ShortUrl} = handle_post(Url);
+        <<"GET">> ->
+            {Status, ShortUrl} = handle_get(Url)
+    end,
+    Body = "{url: \"" ++ ShortUrl ++ "\"}",
```

Comment:
Writing json by hand for something simple is ok, but this starts to get ugly specially if we add more things. use jsx https://github.com/talentdeficit/jsx